man/asymmetric.preference.game.Rd

```diff
@@ -24,7 +24,7 @@ types.}
 types. The matrix has \sQuote{types} rows and columns. When generating
 an undirected graph, it must be symmetric.}
 
-\item{loops}{Logical constant, whether self-loops are allowed in the graph.}
```

Comment:
Should we open an issue about using constant vs scalar consistently? Here I would actually just write logical?